recipes/modshim/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "modshim" %}
+{% set version = "0.2.0" %}
+{% set python_min = "3.9" %}
```

Comment:
This one needs to be set only if the package's min is higher than conda-forge's.

```suggestion
```